eva/executor/create_udf_executor.py

```diff
@@ -118,3 +123,69 @@ def exec(self, *args, **kwargs):
         yield Batch(
             pd.DataFrame([f"UDF {self.node.name} successfully added to the database."])
         )
+
+    def _try_initializing_udf(
+        self, impl_path: str, udf_args: Dict = {}
+    ) -> UdfCatalogEntry:
+        """Attempts to initialize UDF given the implementation file path and arguments.
+
+        Args:
+            impl_path (str): The file path of the UDF implementation file.
+            udf_args (Dict, optional): Dictionary of arguments to pass to the UDF.
+            Defaults to {}.
+
+        Returns:
+            UdfCatalogEntry: A UdfCatalogEntry object that represents the initialized
+            UDF.
+
+        Raises:
+            RuntimeError: If an error occurs while initializing the UDF.
+        """
+        # load the udf class from the file
+        try:
+            # loading the udf class from the file
+            udf = load_udf_class_from_file(impl_path, self.node.name)
+            # initializing the udf class calls the setup method internally
+            udf(**udf_args)
```

Comment:
I'm not sure I completely understnad this. It seems like you are passing in the model as  "yolov8m.pt". But that is also happening inside the lambda function that you are returning. this means we are essentially doing `Yolo("yolov8m.pt")("yolov8m.pt")`?.

Reply:
Which lambda function?

Reply:
Ohh wait my bad, this is loading the UDF from file.